src/bridge_gui.cpp

```diff
@@ -130,6 +130,22 @@ class BuildBridgeWindow : public Window {
 		this->SetWidgetDirty(WID_BBS_BRIDGE_LIST);
 	}
 
+	StringID GetBridgeSelectString(const BuildBridgeData &bridge_data) const {
+		/**
+		* Get the StringID to draw in the selection list and set the appropriate DParams.
+		* @param bridge_data the bridge to get the StringID of.
+		* @return the StringID.
+		*/
+		SetDParam(2, bridge_data.cost);
+		SetDParam(1, bridge_data.spec->speed);
+		SetDParam(0, bridge_data.spec->material);
```

Comment:
Super nitpicky, but do these need to be out of order? (I see that they were originally, but since you're redoing it anyway... 🙂)